Riot/Modules/Room/RoomInfo/RoomInfoList/RoomInfoListViewController.swift

```diff
@@ -159,6 +159,7 @@ final class RoomInfoListViewController: UIViewController {
     // MARK: - Private
     
     private func updateSections(with viewData: RoomInfoListViewData) {
+        isRoomDirect = viewData.isDirect
```

Comment:
This variable and `isDirect` property seem not used. Can we remove it?

Reply:
Thanks for your remark @ismailgulek. I saw you removed this variable in the past, but it is used to compute the text button when you try to leave the room. https://github.com/vector-im/element-ios/blob/e0237d421b00bff4b74223e62e0111503bcd82bf/Riot/Modules/Room/RoomInfo/RoomInfoList/RoomInfoListViewController.swift#L64

Reply:
Ah sorry, it's silly that i thought it was a local variable :)